offline/offline.html

```diff
@@ -0,0 +1,33 @@
+<!doctype html>
```

Comment:
why a html page like this? Would it not be possible to include it in the cms? so the text won't need to be hardcoded?